### PR TITLE
feature(Recipe entity):

### PR DESCRIPTION
**Pull request by rhonniel**
- Recipe is the list of steps for make a dish
- StepRecipe explain how and how much use one ingredient

### CODE REVIEW

File: src/com/mercado/model/product/StepRecipe.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package com.mercado.model.product;
+
+import com.mercado.model.BaseEntity;
+
+public class StepRecipe extends BaseEntity {
```
Comment (reynoldmorel) [inline src/com/mercado/model/product/StepRecipe.java:5]: Maybe `RecipeStep` sounds better?

**Review by reynoldmorel — APPROVED** [review]
I think you might not need this for your first approach since it will be a bit complex without need. But this is just a warning.

**Comment by rhonniel** [comment]
Yeah maybe you are right, but that's the problem when nobody is the client, for example my imaginary client  says 🥸 "  the business need this structure ". 😅

File: src/com/mercado/model/product/StepRecipe.java (hunk 1 above)
Comment (rhonniel) [reply src/com/mercado/model/product/StepRecipe.java:5]: Yeah, you are right.

**Pull request merged**